app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ActiveRecord::Base
   validates :email, presence: true, uniqueness: true
   validates :password_digest, presence: true
+
+  def name
```

Comment:
Cool. I think a slightly better way to do this would be

```ruby
def name
  "#{first_name} #{last_name}"
end
```

Join may work fine, but I find that you sort of have to translate it's meaning in your head. "This method takes all elements of this array, turns them into strings, and separates the strings with a space." As opposed to "Print the first name, then a space, then the last name".